Assets/Engine/Inventory/Editor/ContainerDescriptorEditor.cs

```diff
@@ -359,6 +387,8 @@ private void RemoveSync()
             if (containerDescriptors != null && containerDescriptors.Length == 0)
             {
                 DestroyImmediate(g.GetComponent<ContainerSync>());
+                GameObject networkedParent = GetParentNetworkIdentity(g);
+                DestroyImmediate(networkedParent.GetComponent<ContainerSync>());
```

Comment:
if the GetComponent returns a null, is the DestroyImmediate ok with it? I never trust the results of a GetComponent call :D

Reply:
it's okay with it ! Tested and it handles null without any warning or errors.